src/graph_ops.cc

```diff
@@ -38,39 +141,58 @@ torch::Tensor TensorUnique(torch::Tensor node_ids) {
   }
 }
 
-std::tuple<torch::Tensor, torch::Tensor, torch::Tensor> GraphRelabel(
-    torch::Tensor col_ids, torch::Tensor indptr, torch::Tensor indices) {
-  torch::Tensor frontier, relabeled_indices, relabeled_indptr;
-  std::tie(frontier, relabeled_indices) = impl::RelabelCUDA(col_ids, indices);
-  relabeled_indptr = indptr.clone();
-  return std::make_tuple(frontier, relabeled_indptr, relabeled_indices);
+std::tuple<torch::Tensor, std::vector<torch::Tensor>> BatchTensorRelabel(
```

Comment:
Why we have `BatchTensorRelabel` in this PR? It is not used.

Reply:
`BatchTensorRelabel` is used in `GraphRelabel` now. I will add necessary comments.